spec/index.md

```diff
@@ -3327,6 +3327,19 @@ messages = Older_messages · Younger_messages  ·
 
 ```
 
+But a call to a frozen canister is rejected with `SYS_TRANSIENT` reject code:
```

Comment:
Could we please split up the transition into two transitions tearing
```
S.canister_status[CM.callee] = Stopped or S.canister_status[CM.callee] = Stopping _ or liquid_balance(
```
into
```
S.canister_status[CM.callee] = Stopped or S.canister_status[CM.callee] = Stopping _
```
in one transition and
```
liquid_balance(...
```
in the other one?